### ISSUE TITLE
Possible to export the robot state message from the modern driver

### ISSUE THREAD

**Issue by robotlinker** [ThomasTimm/ur_modern_driver#107]
Hi, I am using the ur_modern_driver to control a UR10 robot manipulator. Before that, I have ever used the old UR driver provided by ROS-I consortium and connect with the real-time data interface using testRT_comm.py to get all robot states output in the /robot_state topic.  
My question is how can I get all real-time data information when I am using the modern driver? Is there any similar python file command or rosrun node to do this? Thanks!

**Comment by gavanderhoorn** [comment]
`ur_modern_driver` does not supply something like `testRT_comm.py` currently I'm afraid.

The infrastructure is all there, but it's C++, and there is no entrypoint that could be used like the script you mention.

It could be added, but as that would be a rather unusual use of the driver, I'm inclined to say it's out-of-scope for now.

Note that you should still be able to use `testRT_comm.py` in parallel to `ur_modern_driver`: the real-time interface supports multiple connections.
